package.json

```diff
@@ -61,7 +62,9 @@
     "@kachkaev/markdownlint-config": "^0.3.0",
     "@netlify/plugin-nextjs": "^4.2.1",
     "@next/eslint-plugin-next": "^12.0.9",
+    "@types/md5": "^2.3.1",
     "@types/node": "^16.11.21",
+    "@types/query-string": "^6.3.0",
```

Comment:
If you decide to keep `query-string` v7, we won’t need external typings: https://www.npmjs.com/package/@types/query-string